networkx/algorithms/tree/tests/test_junction_tree_algorithm.py

```diff
@@ -0,0 +1,79 @@
+import networkx as nx
+from networkx.algorithms.tree.junction_tree_algorithm import junction_tree
+
+
+def test_junction_tree_directed_confounders():
+    B = nx.DiGraph()
+    B.add_edges_from([("A", "C"), ("B", "C"), ("C", "D"), ("C", "E")])
+
+    G = junction_tree(B)
+    J = nx.Graph()
+    J.add_edges_from(
+        [
+            (("C", "E"), ("C",)),
+            (("C",), ("A", "B", "C")),
```

Comment:
I'm confused by this test example. I don't know much about junction trees so I may be missing something.
One node in the junction tree is ```("A","B","C")```. But in this example, A-B-C is not a clique of the original graph.
Also, you have two edges in line 14 and 15 which seem to be opposite direction of the same nodes. J is undirected, so you should only add one of these two lines?   But still, it seems to me that these lines should be:
```(("C",),  ("A", "C")),
      (("C"),  ("B", "C"))
```

Reply:
I see now that the junction tree is not using cliques of the original graph, but cliques of the moralized, triangulated graph.  The docstring should be corrected to accurately describe a junction tree.  :)

Reply:
You're right, I updated the docstring accordingly :)